### ISSUE TITLE
[semantic-issue] Add semantics for sklearn (LogisticRegression) predict 

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#179]
**Description**
The predict method is called on LogisticRegression, GradientBoostingClassifier, SVC objects (Same return value and same lib).

Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
models = {'LG': LogisticRegression(), 'Ridgeclassifier': RidgeClassifier()}
models.fit(x_train, y_train).predict(x_train)
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html

**Note (optional)**
Returns an array of predicted class labels
